hack/init_postgres.sql

```diff
@@ -110,4 +110,6 @@ ALTER table "auth".users OWNER TO supabase_auth_admin;
 ALTER table "auth".refresh_tokens OWNER TO supabase_auth_admin;
 ALTER table "auth".audit_log_entries OWNER TO supabase_auth_admin;
 ALTER table "auth".instances OWNER TO supabase_auth_admin;
-ALTER table "auth".schema_migrations OWNER TO supabase_auth_admin;
\ No newline at end of file
+ALTER table "auth".schema_migrations OWNER TO supabase_auth_admin;
+ALTER FUNCTION auth.uid OWNER TO supabase_auth_admin;
+ALTER FUNCTION auth.role OWNER TO supabase_auth_admin;
```

Comment:
looks like we need to add `auth.email()`?

Reply:
also, does this script run after the functions are created? will this work if the functions are not created first?